image_width: 640
image_height: 480
camera_name: narrow_stereo
camera_matrix:
  rows: 3
  cols: 3
  data: [1045.26332,    0.     ,  316.10259,
            0.     , 1045.06462,  211.27367,
            0.     ,    0.     ,    1.     ]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [0.248491, -1.392918, -0.017943, -0.002283, 0.000000]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1., 0., 0.,
         0., 1., 0.,
         0., 0., 1.]
projection_matrix:
  rows: 3
  cols: 4
  data: [1061.81948,    0.     ,  315.27759,    0.     ,
            0.     , 1056.3245 ,  207.51142,    0.     ,
            0.     ,    0.     ,    1.     ,    0.     ]

